### ISSUE TITLE
Clusters changed for assigned text after reducing outliers

### ISSUE THREAD

**Issue by lysummer55** [MaartenGr/BERTopic#1520]
Hi,

I was calling 
# Train your BERTopic model
topic_model = BERTopic()
topics, probs = topic_model.fit_transform(docs)

# Reduce outliers
new_topics = topic_model.reduce_outliers(docs, topics)

to reduce outliers according this page. But I found some of the assigned clusters (before reducing outliers) of the text are changed to a different cluster after reducing outliers.

Shouldn't the already assigned clusters will not change after reducing outliers according to "You can pass outlier and non-outlier documents together since it will only try to reduce outlier documents and label them to a non-outlier topic."? 

Is this an expected phenomenon due to that I misunderstand anywhere or there is something wrong inside it?

Thank you for helping me out!

**Comment by MaartenGr** [comment]
That should indeed be the case when you use `.reduce_outliers`. For completeness, do you have a reproducible example?
